corehq/apps/auditcare/middleware.py

```diff
@@ -107,3 +69,19 @@ def process_response(self, request, response):
             audit_doc.status_code = response.status_code
             audit_doc.save()
         return response
+
+    def should_audit(self, view_func):
+        fn = view_func if isinstance(view_func, FunctionType) else view_func.__class__
+        fqview = f"{view_func.__module__}.{fn.__name__}"
```

Comment:
I know it was just moved, but do you know what `fqview` stands for and is there a better name we could use instead?